buildSrc/src/main/java/org/opensearch/gradle/tar/SymbolicLinkPreservingTar.java

```diff
@@ -219,7 +227,7 @@ private void handleProcessingException(final FileCopyDetailsInternal details, fi
         }
 
         private long getModTime(final FileCopyDetails details) {
```

Comment:
I haven't tested this at all and don't know if this is a good idea, but how about replacing the last modified time with the creation time of the file? It would avoid having to call out to git and maybe make this change even simpler.

```
private long getModTime(final FileCopyDetails details) {
    if (isPreserveFileTimestamps) {
        try {
            final BasicFileAttributes attributes =
                Files.readAttributes(Paths.get(details.getPath()), BasicFileAttributes.class);
            return attributes.creationTime().toMillis();
        } catch (IOException e) {
            return 0;
        }
    }
    return details.getLastModified();
}
```

Reply:
Oh right, of course. For some reason I was thinking that the creation time would be preserved from the original file, but that doesn't even make sense for the newly created files in the build process. Sorry.

Reply:
Np at all